merge_test.go

```diff
@@ -116,4 +151,11 @@ func TestMergeConfig_ToString(t *testing.T) {
 	cypher, err = t6.ToString()
 	req.Nil(err)
 	req.EqualValues("test ON CREATE SET test = $props ON MATCH SET test = $props", cypher)
+
+	//merge with on create and on match with members
+	cypher, err = t7.ToString()
+	req.Nil(err)
+	req.Contains(cypher, "test ON CREATE SET test = $props")
+	req.Contains(cypher, " ON MATCH SET ", " SET test.key1 = 1", " SET test.key2 = 'value2'", " SET test.key3 = $key3")
+	req.Equal(106, len(cypher))
```

Comment:
Also, it looks like this is using the `SET` keyword multiple times, instead of using a comma to separate each value

e.g.
`SET test.key1 = 1 SET test.key2 = 'value2' SET test.key3 = $key3`
instead of 
`SET test.key1 = 1, test.key2 = 'value2', test.key3 = $key3`

It looks like you might be following a convention used elsewhere, but it looks like using a single `SET` keyword may be more efficient

![Screen Shot 2023-01-24 at 9 37 22 AM](https://user-images.githubusercontent.com/8619705/214352929-423d7022-abd4-4f33-9aa2-d721d3440b9e.png)
vs
![Screen Shot 2023-01-24 at 9 36 43 AM](https://user-images.githubusercontent.com/8619705/214352761-f59822c1-d3ee-43c0-93e3-07308702fbf8.png)

Reply:
> Why is this using multiple Contains statements + an Equal statement instead of a single EqualValues like the rest of this file?

As you can see in this PR, the `members` field in the `MergeSetConfigWithMembers` struct is a map. Since the order when iterating the elements of a map is undetermined, using `Contains` will prevent a flaky test instead of asserting the full cypher.

> It looks like you might be following a convention used elsewhere, but it looks like using a single SET keyword may be more efficient

I am using the pattern that [we are currently using in the codebase](https://github.com/SGNL-ai/sgnl/blob/3600d34/pkg/ingestion/workers/query.go#L27), since following existing patterns makes the new code easier to reason about. In addition, the new changes introduced should be much less performance-sensitive than ^.

This being said, a separate story can be created to overhaul the existing hard-coded queries to some more efficient form, if the benefit is statistically significant.

Reply:
nvm, I've updated the code to only have one `SET` since it is a fairly trivial change, together with the renaming change.